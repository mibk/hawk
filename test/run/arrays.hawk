BEGIN {
	x = []
	y[0] = 3
	z[] = 4
	print y[0], z[0], z
	arr = ["January", "February", 1 + 2]
	print arr[0], arr[2 - 1], arr[2], arr[3]
	arr[0] = "Jan"
	print arr[0]
	arr[] = "April"
	print arr[len(arr)-1]
	print arr
	printf "%#v\n", arr
}
