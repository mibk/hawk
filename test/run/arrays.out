3 4 [4]
January February 3 false
Jan
April
["Jan", "February", 3, "April"]
["Jan", "February", 3, "April"]
